### PR TITLE
Fix checkstyle

### PR DESCRIPTION
**Pull request by clayreimann**

### CODE REVIEW

File: elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java [hunk 1]
```diff
@@ -59,6 +59,8 @@
 class EntityBinding {
 
     private static final List<Method> OBJ_METHODS = Arrays.asList(Object.class.getMethods());
+    private static final List<Class<? extends Annotation>> RELATIONSHIP_TYPES =
```
Comment (DeathByTape) [inline elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java]: Should we use a set here and just perform a `contains`?

Iterating and comparing a constant list of 4 may be faster than the constant time hashing, though. @wcekan thoughts here?

File: elide-core/src/main/java/com/yahoo/elide/core/pagination/Pagination.java [hunk 2]
```diff
@@ -193,6 +163,41 @@ private Pagination evaluate(int defaultLimit, int maxLimit) {
         return this;
     }
 
+    private boolean hasInvalidCombiation(Map<PaginationKey, Integer> pageData) {
```
Comment (DeathByTape) [inline elide-core/src/main/java/com/yahoo/elide/core/pagination/Pagination.java]: Typo. `Combiation` -> `Combination`

**Comment by DeathByTape** [comment]
Can we add these change descriptions to the change log?

File: elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java (hunk 1 above)
Comment (wcekan) [inline elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java]: This is fine.  We loop over this list, but do not call `contains`.

**Pull request merged**